documentation/book/ref-storage-jbod.adoc

```diff
@@ -34,10 +40,13 @@ storage:
 # ...
 ----
 
-It is not possible to change the `id` property for a volume.
-It is not possible to add or remove volumes from a JBOD storage configuration.
+Do not change the `id` property of a JBOD volume.
+
+NOTE: Adding and removing volumes from a JBOD configuration is not currently supported.
+
+== JBOD and Persistent Volume Claims
 
-When Persistent storage is used for declaring volumes, the naming schema for the created Persistent Volume Claims is the following:
+When persistent storage is used to declare JBOD volumes, the naming scheme of the resulting Persistent Volume Claims is as follows:
 
-`data-_id_-_cluster-name_-kafka-_idx_`::
-Persistent Volume Claim for the volume `_id_` used for storing data for the Kafka broker pod `_idx_`.
+`data-_id_-cluster-name-kafka-_idx_`::
```

Comment:
the "cluster-name" part is variable as id and idx should be used "_" as I did ?

Reply:
Oops, sorry about that. I will update.